.ci/install_kata_kernel.sh

```diff
@@ -55,6 +55,12 @@ build_and_install_kernel() {
 	"${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "setup"
 	"${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "build"
 	sudo -E PATH="$PATH" "${kernel_packaging_dir}/build-kernel.sh" -v "${kernel_version}" ${extra_opts} "install"
+
+	### REMOVE after kata-containers PR #5010 is merged
+	if [ "${kernel_type}" == "sev" ]; then
+		sudo ln -sf "vmlinux-sev.container" "/usr/share/kata-containers/vmlinux.container"
```

Comment:
somewhat surprised to see the uncompressed kernel here.

Reply:
Not actually using it. Temporary hacky workaround for https://github.com/kata-containers/kata-containers/issues/5008.

Reply:
@ryansavino  now that PR 5013 is merged can we remove this?